ansible/roles/software/ckan/inventory/tasks/main.yml

```diff
@@ -100,15 +100,17 @@
   template: src={{ item }} dest=/{{ item }} owner=root group=www-data mode=0640
   with_items:
     - etc/ckan/production.ini
-    - etc/ckan/apache.wsgi
     - etc/ckan/datapusher.wsgi
     - etc/ckan/datapusher_settings.py
     - etc/apache2/sites-enabled/ckan.conf
     - etc/apache2/sites-enabled/datapusher.conf
     - etc/supervisor/conf.d/gunicorn-web-app.conf
-  notify:
-    - restart apache2
-    - reload supervisor  # TODO the supervisor config files should be in their own task
+  notify: restart apache2
```

Comment:
If `gunicorn-web-app.conf` is modified, don't you want to notify supervisor about the change?